test/acceptance/integration/Rules/Cards/card_create_rules.feature

```diff
@@ -171,8 +171,8 @@ Feature: Append a new rule in Perseo manager using cards from portal
     | test_900003 | room_*      | updateAttribute  | DANGER                                                | ALARM           |
     | test_900004 | room.*      | SendSmsMibAction | ${device_latitude}${device_longitude}${measure.value} | 123456789       |
 
-  @not_updated_card_without_card
-  Scenario: append a new rule in Perseo manager using card from portal using only not updated card without action card
+  @not_updated_card_without_AC
```

Comment:
AC could be mislead with "Access Control"... I would suggest for using the expanded name (action_card)